packages/button/Button.tsx

```diff
@@ -1,33 +1,44 @@
-import { h, Component, prop } from 'skatejs';
+import { h, Component, prop, ComponentProps } from 'skatejs';
 
-import { ColorType, cssClassForColorType, css } from '../_helpers';
+import {
+  ColorType,
+  cssClassForColorType,
+  css,
+  IntrinsicCustomElement,
+  IntrinsicBoreElement,
+  ClickEvent
+} from '../_helpers';
 
 import styles from './Button.scss';
 
-// public
-type ButtonProps = Props & EventProps;
-type Props = {
+export type ButtonProps = Props & Attrs & EventHandlers;
+export type Attrs = {
```

Comment:
we could probably have everything in `Props` as all props will have `attribute:{source: true}` as default right? Or is there any reason to explicitly state it.